ui/src/App.scss

```diff
@@ -24,7 +24,7 @@
 
     // :fist-child to avoid setting padding for `UserFeedback`
     & > *:first-child:not(.__Jobs__) {
-      @media (min-width: $screen-xs-max) {
+      @media (min-width: $screen-lg-max) {
```

Comment:
?

Reply:
I discussed with Stella about removing the margin in the sides for tablets so that the graph numbers don't overlap.

Reply:
we shouldn't remove those paddings in whole pages just because of a graph. I'd say leave this. And I am already playing with those in #800, we can decide later if that's really needed or not.

@StellaCh